precompose/src/commonMain/kotlin/moe/tlaster/precompose/navigation/BackStackManager.kt

```diff
@@ -88,9 +88,9 @@ internal class BackStackManager(
             options != null &&
             matchResult.route is SceneRoute &&
             options.launchSingleTop &&
-            _backStacks.any { it.hasRoute(matchResult.route.route) }
+            _backStacks.any { it.hasRoute(matchResult.route.route, path, options.includePath) }
         ) {
-            _backStacks.firstOrNull { it.hasRoute(matchResult.route.route) }?.let {
+            _backStacks.firstOrNull { it.hasRoute(matchResult.route.route, path, options.includePath) }?.let {
```

Comment:
im not quite sure if this check should also be considering `includePath` in the PopUp case at L118

Reply:
I thinks it's better to check `includePath' at L118 to maintain consistent behaviour.

Reply:
consider it done